core/src/main/scala/pureconfig/syntax/package.scala

```diff
@@ -19,12 +19,11 @@ package object syntax {
 
   implicit class PimpedConfigValue(val conf: ConfigValue) extends AnyVal {
     def to[T](implicit configConvert: ConfigConvert[T]): Either[ConfigReaderFailures, T] = configConvert.from(conf)
-    def toOrThrow[T](implicit configConvert: ConfigConvert[T], cl:ClassTag[T]): T = getResultOrThrow(configConvert.from(conf))(cl)
+    def toOrThrow[T](implicit configConvert: ConfigConvert[T], cl: ClassTag[T]): T = getResultOrThrow(configConvert.from(conf))(cl)
```

Comment:
We have an issue with code non-formatted when it's submitted, we should fix this eventually.